openmetadata-ui/src/main/resources/ui/playwright/e2e/Features/ActivityFeed.spec.ts

```diff
@@ -395,177 +394,170 @@ test.describe('Mention notifications in Notification Box', () => {
     }
   });
 
-  test.afterAll('Cleanup entities and users', async ({ browser }) => {
-    const { apiContext, afterAction } = await performAdminLogin(browser);
-
-    try {
-      await entity.delete(apiContext);
-      await user1.delete(apiContext);
-      await adminUser.delete(apiContext);
-    } finally {
-      await afterAction();
-    }
-  });
-
   test('Mention notification shows correct user details in Notification box', async ({
     browser,
   }) => {
-    const { page: adminPage, afterAction: afterActionAdmin } =
-      await performUserLogin(browser, adminUser);
-    const { page: user1Page, afterAction: afterActionUser1 } =
-      await performUserLogin(browser, user1);
+    const adminPage = await browser.newPage();
+    await adminUser.login(adminPage);
 
-    await test.step(
-      'Admin user creates a conversation on an entity',
-      async () => {
-        await entity.visitEntityPage(adminPage);
+    const user1Page = await browser.newPage();
+    await user1.login(user1Page);
 
-        await adminPage.getByTestId('activity_feed').click();
-        await adminPage.waitForLoadState('networkidle');
-
-        await adminPage.waitForSelector('[data-testid="loader"]', {
-          state: 'detached',
-        });
-
-        await adminPage.getByTestId('comments-input-field').click();
-
-        await adminPage
-          .locator(
-            '[data-testid="editor-wrapper"] [contenteditable="true"].ql-editor'
-          )
-          .fill('Initial conversation thread for mention test');
-
-        await expect(
-          adminPage.locator('[data-testid="send-button"]')
-        ).toBeVisible();
-        await expect(
-          adminPage.locator('[data-testid="send-button"]')
-        ).not.toBeDisabled();
+    try {
```

Comment:
Check out the `fixtures` for playwright. We should avoid try-catch for such things. We have some tests already using the fixtures.